learn/get-started/build-a-data-service-in-ballerina.md

```diff
@@ -471,15 +481,16 @@ bal run
 You view the output below.
 
 >**Info:** This creates an `/employees` endpoint on port `8080`, which can 
-be accessed via a browser by visiting `http://locahost:8080/employees`.
+be accessed via a browser by visiting `http://localhost:8080/employees`.
 
 ## Try the service
 
 Invoke the defined resource function by sending the `POST` request below to `http://localhost:8080/employees` with the required data as a JSON payload.
 
+>**Info:** The following CURL command only works with Unix operating systems.
```

Comment:
```suggestion
>**Info:** The following cURL command only works with Unix operating systems.
```
Can we give the command for Windows as well in the info section saying, "If you are using the Windows OS, execute this cURL command: <command>."